composer.json

```diff
@@ -24,7 +20,7 @@
         "phing/phing": "^2.16",
         "behat/mink-extension": "^2.2",
         "behat/mink-selenium2-driver": "^1.3",
-        "centreon/centreon-test-lib": "dev-master",
+        "centreon/centreon-test-lib": "dev-MON-2771-change-templates-to-select2",
```

Comment:
to remove before merging

Reply:
if you've already builded your branch on jenkins and acceptance tests passed, you don't need this anymore. Please remove it...